src/sql/workbench/browser/modal/modal.ts

```diff
@@ -432,6 +432,16 @@ export abstract class Modal extends Disposable implements IThemable {
 				dialogWidth = this._modalOptions.width;
 			}
 
+			if (this._modalOptions.dialogPosition === 'above') {
+				if (this._modalOptions.dialogProperties) {
+					this._modalDialog.style.left = `${this._modalOptions.dialogProperties.xPos - this._modalOptions.dialogProperties.width}px`;
+					this._modalDialog.style.top = `${this._modalOptions.dialogProperties.yPos - 235}px`;
```

Comment:
What does 235 refer to?

Reply:
the height of the dialog.. let me try to find that property

Reply:
The height of the dialog is not stored anywhere right now - only the max-height is stored:
https://github.com/microsoft/azuredatastudio/blob/b1daa4ee4810393b2d03583f646400c2c1695149/src/sql/workbench/browser/modal/media/calloutDialog.css#L12
Given that @halerankin will be refactoring the callout code, is it alright if I leave this for now?